### ISSUE TITLE
group-contract : Invalid code when using multiple cardinality of group within a sequence

### ISSUE THREAD

**Issue by trazere** [mklemm/jaxb2-rich-contract-plugin#16]
Hello

When using group-contract, the code generated by this fragment of schema does not compile :

```
	<group name="MyGroup">
		<sequence>
			<element name="E1" type="string" />
			<element name="E2" type="int" />
		</sequence>
	</group>

	<complexType name="Ok">
		<group ref="tns:MyGroup" />
	</complexType>

	<complexType name="OkToo">
		<sequence>
			<group ref="tns:MyGroup" />
		</sequence>
	</complexType>

	<complexType name="OkAgain">
		<group ref="tns:MyGroup" minOccurs="0" maxOccurs="unbounded" />
	</complexType>

	<complexType name="Ko">
		<sequence>
			<group ref="tns:MyGroup" minOccurs="0" maxOccurs="unbounded" />
		</sequence>
	</complexType>
```

Compile errors :
- The type Ko must implement the inherited abstract method MyGroup.setE2(int)
- The type Ko must implement the inherited abstract method MyGroup.setE1(String)
- The type Ko must implement the inherited abstract method MyGroup.getE2()
- The type Ko must implement the inherited abstract method MyGroup.getE1()

**Comment by mklemm** [comment]
You're right, this case wasn't tested. This should be fixed in the next release...

**Comment by trazere** [comment]
Thanks a lot !

**Comment by mklemm** [comment]
This issue has been fixed in release 1.6.4.
